javascript/main.js

```diff
@@ -0,0 +1,12 @@
+let feeTotal = 0;
+
+function alertDogInfo(name, breed, adoptionFee) {
+  let txt =
+    'Name: ' + name + '\nBreed: ' + breed + '\nAdoption fee: ' + adoptionFee;
+  alert(txt);
+}
+
```

Comment:
Good work. You might also be interested in template literals for string concatenation: 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

I find the syntax to be a little cleaner.